images-optimize-and-upload-cf7/frontend/assets/script.js

```diff
@@ -100,6 +101,31 @@ jQuery(document).ready(function ($) {
         errorHandler() //hide error
         countImages = 0
       }, false);
+
+      // callback invalid input
+      document.addEventListener('wpcf7invalid', function (event) {
+        const uploadErr = arrayObjHasPropertyValue(event.detail.apiResponse.invalid_fields, 'field', NAME_TAG);
```

Comment:
There may be an error here, an undefined property.
event.detail.apiResponse.invalid_fields